CHANGELOG.md

```diff
@@ -5,6 +5,25 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+## [1.1.2-alpha.0] - 2023-10-03
+
+### Added
+
+- Vendorized [bootstrap-vue](https://github.com/bootstrap-vue/bootstrap-vue)
+  SCSS and rewrote using modular SCSS
+
+### Changed
+
+- Upgraded `es-bs-base` SASS compiler from `node-sass` to `dart-sass`
+- Refactored all SCSS files to use `@use` or `@forward` instead of the
+  deprecated `@import` feature
```

Comment:
One of these is the [breaking](https://github.com/vweevers/common-changelog#244-prefixes) change.